packages/fx-core/resource/package.nls.json

```diff
@@ -226,6 +226,8 @@
   "core.DashboardOption.detail": "A canvas with cards and widgets for displaying important information",
   "core.BotNewUIOption.label": "Basic Bot",
   "core.BotNewUIOption.detail": "A simple implementation of a bot that's ready to customize",
+  "core.LinkUnfurlingOption.label": "Unfold a Link into Card",
+  "core.LinkUnfurlingOption.detail": "Expand the links to show additional information in Microsoft Teams and Outlook",
```

Comment:
"Display information and actions when a URL is pasted into the compose message area."